### ISSUE TITLE
Jerome Saint, Epistulae - in Perseus the table of contents contains the name of the letters, not just a number

### ISSUE THREAD

**Issue by monzug** [alpheios-project/alpheios_nemo_ui#12]
in Perseus, the Jerome Saint's Epistulae table of Contents is listed by the name of the letters
e.g.  Ad Innocentium Presbyterum De Septies Percussa  in Perseus is equal to sections 1.1-1.5  and 1.6-1.10 and 1.11-1.15 in Alpheios.
We do have <head>Ad Innocentium Presbyterum De Septies Percussa</head>
I would add the name of the letters.
2) Ad Chromatium, Iovinum, Eusebium  = 7.1-7.5  7.6
<head> Ad Chromatium, Iovinum, Eusebium</head>
3) Ad Heliodorum Monachum = 14.1-14.5 
<head> Ad Heliodorum Monachum</head> = 14.1-14.5 14.6-14.10 14.11
4) An Eustochium  = 22.1-22.5, 22.6-22.10, 22.11-22.15, 22.16-22.20,22.21-22.25,22.26-22.30...till 22.41
<head> An Eustochium</head>
5) Ad Marcellam = 38.1-38.5
<head> Ad Marcellam</head>
6) Ad Marcellam De Onaso = 40.1-40.3
<head> Ad Marcellam De Onaso</head>
7) Ad Marcellam =43.1-43.3
<head> Ad Marcellam</head>

8) <head> Ad Marcellam</head>  we are missing to display this section!!!!!!!!!!!
missing in Alpheios and in Scaife but we have the content in the XML. it's present in Perseus

9) Ad Asellam 
<head> Ad Asellam</head> = 45.1-45.5, 45.6-45.7
10) Ad Nepotianum Phesbyterum
<head> Ad Nepotianum Phesbyterum</head> = 52.1-52.5, 52.6-52.10,...52.17
11)  Ad Furiam De Viduitate Seruanda 
<head> Ad Furiam De Viduitate Seruanda</head>  = 54.1  to  54.18
12) Ad Heliodorum Epitaphium Nepotiani 
<head> Ad Heliodorum Epitaphium Nepotiani</head>  =  60.1-60-5 to 60.19
13) Ad Oceanum De Morte Fabiolae 
<head> Ad Oceanum De Morte Fabiolae</head>  =  77.1-77.5 to 77.12
14)  Ad Laetam De Institutione Filiae 
<head> Ad Laetam De Institutione Filiae</head>  =  107.1-107.5 to 107.13
15)  An Matrem et Filiam In Gallia Commorantes 
<head> An Matrem et Filiam In Gallia Commorantes</head> =  117.1-117.5 to 117.12
16)  Ad Rusticum Monachum 
<head> Ad Rusticum Monachum</head>  =  125.1-125.5 to 125.20
17)  Ad Principiam Virginem De Vita Sanctae Marcellae 
<head> Ad Principiam Virginem De Vita Sanctae Marcellae</head>  =  127.1-127.5 to 127.14
18) Ad Pacatulam
<head> Ad Pacatulam</head>   =  128.1-128.5

also, we are missing the publisher and I believe this has been reported already in #8 
e.g. <monogr><title>Select Letters of St. Jerome</title><author>Jerome, Saint</author><editor>F.A. Wright</editor><imprint><publisher>William Heinemann Ltd.; Harvard University Press</publisher><pubPlace>London; Cambridge, Massachusetts</pubPlace><date>1933</date></imprint></monogr>

to sum up, 3 issues here:
missing the name of the letters
missing to display some content
missing the pubblisher

**Comment by balmas** [comment]
(1) is probably the same issue as #23 -- the titles of the sections are outside of the citation scheme hierarchy in the markup so they don't display
(2) letter 44 doesn't have any sections, so it's not adhering to the citation scheme for the text. Both a data issue and a CTS implementation limitation.
(3) missing the publisher: this is both a data and code question. The publisher is not included in the metadata file right now, which is why it isn't displayed.  Ideally the application would pull metadata from the teiHeader itself, this isn't implemented yet in the underlying framework. Related issues: https://github.com/Capitains/MyCapytain/issues/148 and https://github.com/Capitains/guidelines/issues/3
